lib/format/datetime/formatter.ex

```diff
@@ -80,7 +87,7 @@ defmodule Timex.Format.DateTime.Formatter do
       try do
         {:ok, lformat!(date, format_string, locale, formatter)}
       catch
-        _type, %FormatError{:message => msg} ->
```

Comment:
I think you need to handle both errors here

Reply:
`%FormatError{:message => msg}` will match with `%{:message => msg}`, since processing of them is same I decided to remove this case